### PR TITLE
Create a last_login test data seeder

### PR DESCRIPTION
**Pull request by MKodde**
This seeder can create a given amount of last_login entries. A date offset must be specified to control the date of the users last login time.

This script is usefull when you want to simulate larger batch deprovision actions. But note that the actual users probably do not exist on the platform.

### CODE REVIEW

File: bin/seed_test_users.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+if ! [[ $1 =~ $numberExpression ]] ; then
+   echo "Usage: The first argument must be a number. Indicating the number of identities you'd like to seed to the last_login table" >&2; exit 1
+fi
+
+if ! [[ $2 =~ $numberExpression ]] ; then
+   echo "Usage: The second argument must be a number. Indicating the number of days since last login for all users" >&2; exit 1
+fi
+
+if ! [ -x "$(command -v sqlite3)" ]; then
+    echo "The SQLite binary is not available"
```
Comment (quartje) [inline bin/seed_test_users.sh]: You might want to exit here if the binary is not available

File: bin/seed_test_users.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+fi
+if ! [ -x "$(command -v rig)" ]; then
+    echo "The rig binary is not available, Rig is used to create a random username"
```
Comment (quartje) [inline bin/seed_test_users.sh]: Same here: You might want to exit here. You could also use already available commands to generate something random, like:
` head /dev/urandom |  tr -dc A-Za-z0-9 | head -c 13 ; echo '`

File: bin/seed_test_users.sh [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+fi
+
+if ! [ -f "${databasePath}" ]; then
+    echo "The SQLite database was not found at '${databasePath}'"
+fi
+
+echo "Adding ${1} users to the last_login test table.."
+for ((i = 1 ; i <= $1 ; i++)) do
+    randomUserName=`rig | head -n 1 |  tr ' ' _ | tr A-Z a-z`
+    timeLastLogin=`date --rfc-3339=seconds -d "-${2} days"`
```
Comment (quartje) [inline bin/seed_test_users.sh]: The database that we use in production has the format "YYYY-mm-dd":
MariaDB [log_logins_prd]> select * from last_login limit 1;
+------------------------------------+------------+
| userid                             | lastseen   |
+------------------------------------+------------+
| urn:collab:person:example.com:1234 | 2019-02-15 |
+------------------------------------+------------+

File: bin/seed_test_users.sh (hunk 1 above)
Comment (MKodde) [reply bin/seed_test_users.sh]: Yes, completely missed that :ok_hand:

File: bin/seed_test_users.sh (hunk 2 above)
Comment (MKodde) [reply bin/seed_test_users.sh]: Leaving this as-is for now. If ever RIG starts working against us (as in it becomes difficult to install on a test/dev rig) I will get rid of it then.

File: bin/seed_test_users.sh (hunk 3 above)
Comment (MKodde) [reply bin/seed_test_users.sh]: Cool, changed the format to `date`

**Comment by MKodde** [comment]
Thanks for your valuable review feedback @quartje. I've addressed your feedback, feel free to re-review.

**Pull request merged**